CMakeLists.txt

```diff
@@ -163,78 +163,6 @@ endif()
 set (VERSION_STRING "${DYNINST_MAJOR_VERSION}.${DYNINST_MINOR_VERSION}.${DYNINST_PATCH_VERSION}")
 set (DYNINST_NAME "DyninstAPI-${VERSION_STRING}")
 
-if(BUILD_TARBALLS)
```

Comment:
This giant big beautiful block of red deletion is very satisfying, +1 destroy it.